### DISCUSSION TITLE
How to actually manage the boiler?

### DISCUSSION THREAD

**Discussion by maddhin** [emsesp/EMS-ESP32#689] [General]
Hi,
I do have a naive but elementary question: what is actually the best/design way of managing the boiler / heating?

What I mean is, how the "heating" (as general term) is supposed to be controlled. 

I do have a Junkers boiler and a CR100 control as well as Homematic IP thermostats in each room. As the CR100 control uses the temperature of the hallway, I have a problem when I e.g. want the living room at 23 degree but the hallway is at 20 degree = the shut off temp of the boiler. So the living room stays "cold". 

IMHO there are now 3 different ways of solving the problem:
1. set the temp-desired auf the CR100 to, say, 23 degree so the boiler starts and the living room gets warm.
2. start the boiler directly, i.e. e.g. setting the flow temp (?) 
3. set up some - if technically feasible - virtual controler ("virtual CR100") which - if technically feasible - then is fed with the desired temps and starts the boiler. And the real CR100 is then set e.g. to 10 degress only as a backup. 

So, from the ESM_ESP point of view, which is the best / designed way to deal with this? Is there any best practice solutions which then can be translated in the respective home automatisation server (I'm using FHEM).

Sorry for this question but I found a lot of technical advise but failed to find an answer to this question. 

Many thanks!

**Comment by proddy** [comment]
good question, I'm not an expert though. @bbqkees?

**Comment by MichaelDvP** [comment]
You can not control flowtemp if you have a thermostat, it will always overwrite your setting. You can only control the thermostat by setting the desired roomtemp. 
The thermostat has to be mounted in a reference room with max heat demand and full open radiator valves. All other rooms can be reduced by the radiator valves. The other way is to a thermostat with outdoor temperature control (heating curve). This calculaes the desired flowtemp and all rooms can use radiator valves.

**Comment by maddhin** [comment]
So, this basically means I either use the existing thermostat to control the boiler or I disconnect the thermostat physically (e.g. add a switch) and then completely control the boiler through EMS-ESP / the smart home server? No 3rd way?

**Reply by proddy** [reply]
Can't think of a 3rd way

**Reply by wimvermoens** [reply]
Hello. I have a question that is in the same theme. Please let me know if it is better to open a new thread.

I have a thermostat CW-400 "that is mounted in between of the two knobs, clicked onto the junkers cerapur". So the measured temperature is ignored, and so it is fully depending on the external measured temperature that is connected to the system.

I have connected the gateway s32 to it, and I can see both the heater and the CW-400 from the bus system.

Also works with a Loxone domotica system, and every room has its own actuator valve, and its own room temperature measured with 1-wire temp sensors.

On the collector there is a bypass so if all valves are closed the system will not block the pump.
Furhtermore there is an external DHW junkers storcell.

How do I need to manage my system to reduce gas consumption:
- If we are "away" I want to stop the DHW to keep its temperature. Offcourse I have to desinfect it afterwarths before use.
- If there is no heat needed, the heating needs to be stopped, if there is a demand of heat it needs to be activated. 
- the loxone system calculates a wanted flow temperature, is this possible to set, because of the CW-400 is just sitting on top of the junkers?

In the future I will add a Heat Storage Tank to my system, to heat the returning water.
- Can I activate the internal circulation pump if the returning water temp is higher than the calculated flow temperature?
- Does the junkers allows this kind of negative delta t?
- If not, I guess I need to add another circulation pump?

Thanks in advance!
Keep up the good work!
